samples/getting-started/azure-quantum/README.md

```diff
@@ -0,0 +1,61 @@
+---
+page_type: sample
+languages:
+- python
+products:
+- qdk
+- azure-quantum
+description: "This sample demonstrates how to use Honeywell and IonQ on the Azure Quantum service with quantum circuits expressed in various Python libraries."
+urlFragment: azure-quantum-with-python-circuits
+---
+
+# Getting started with Python libraries on Azure Quantum
+
+These sample notebooks demonstrate how to use the `azure-quantum` Python library to submit quantum circuits expressed with [Cirq](https://quantumai.google/cirq), [Qiskit](https://github.com/QISKit/qiskit-terra) and the provider-specific formats [OpenQASM 2.0](https://github.com/Qiskit/openqasm/tree/OpenQASM2.x) and [JSON](https://docs.ionq.com/#tag/quantum_programs) to an Azure Quantum Workspace.
```

Comment:
Nit, but if I understand IonQ's documentation correctly, they accept their JSON schema and not JSON in general?

Reply:
Correct.